app/controllers/curation_concerns/curation_concerns_controller.rb

```diff
@@ -43,7 +54,15 @@ def presenter
     end
 
     def manifest_builder
-      ManifestBuilder.new(presenter, ssl: request.ssl?)
+      ManifestBuilder.new(presenter, ssl: request.ssl?, login_url: login_url, logout_url: logout_url)
```

Comment:
Or even just services: [auth_service], so we leave it open for more services on the manifest.